caravel/config.py

```diff
@@ -113,7 +113,7 @@
 LANGUAGES = {
     'en': {'flag': 'us', 'name': 'English'},
     # 'fr': {'flag': 'fr', 'name': 'French'},
-    # 'zh': {'flag': 'cn', 'name': 'Chinese'},
+    'zh': {'flag': 'cn', 'name': 'Chinese'},
```

Comment:
Is the website mostly translated at this point? I feel like we may want to leave the default to `en` only until we have really good translation coverage.